readme-files/OpcUa-Server-Stacks-Features.md

```diff
@@ -0,0 +1,30 @@
+# OPC UA Server Stacks Comparison
+
+The following matrix shows a comparison about the features of common OPC UA server stacks. The comparison was done in December 2021 with the focus on higher-level languages. For an up to date list of supported features, please refer to each individual OPC UA stack repository.
+
+Cells with a question mark we could not determine exactly. If you have information on these, please create a pull request.
+
+<br>
+
+|            | NodeOPCUA | Eclipse Milo | OPCFoundation |
+| :--------- | :----- | :----------- | :------------ |  
+| Repository Link | [NodeOPCUA](https://node-opcua.github.io/) | [Eclipse](https://github.com/eclipse/milo) | [UA-.NETStandard](https://github.com/OPCFoundation/UA-.NETStandard) |
+| Programming language | JavaScript, TypeScript | Java | C#  |
+| Supports OPC UA specification 1.03 (*Published 2015*) | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| Supports OPC UA specification 1.04 (*Published 2017*) | :grey_question: | :x: | :heavy_check_mark: |
+| Import of NodesSet2.xml | :heavy_check_mark: | :x: | :x: |
+| Proprietary xml import | :x: | :x: | :heavy_check_mark: (predefined xml output from model compiler) |
```

Comment:
node-opcua: Importing of instance XML are possible see for example here: https://github.com/AndreasHeine/SampleServer-node-opcua